solutions/swb-app/src/sshKeys/createSshKeyResponse.ts

```diff
@@ -10,7 +10,7 @@ export const CreateSshKeyResponseParser = z
   .object({
     projectId: z.string(),
     privateKey: z.string(),
-    sshKeyId: z.string(),
+    id: z.string(),
```

Comment:
wait wasn't it the `beforeEach` that broke the tests??? 😉

Reply:
Thank you for your help figuring out the true root cause!